elements/pfe-icon/demo/pfe-tab.css

```diff
@@ -0,0 +1,461 @@
+    }
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="true"]){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+    --pfe-tabs__tab--BorderLeft:  #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(highlight)};
+    padding-left:                 calc(#{pfe-var(container-padding)} * 1.5 - 2px);
+    @include browser-query(ie11) {
+      border-right:  0;
+      border-left:   #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(highlight)};
+      padding-left:  10px;
+      border-bottom: 0 !important;
+    }
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]:hover){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+    --pfe-tabs__tab--BorderBottom: 0;
+    --pfe-tabs__tab--BorderLeft: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    padding-left: calc(#{pfe-var(container-padding)} * 1.5 - 2px);
+    @include browser-query(ie11) {
+      border-right:  0;
+      border-bottom: 0;
+      padding-left:  10px;
+      border-left:   #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    }
+  }
+}
+
+/// ===========================================================================
+/// WIND TAB VARIANTS / saturated
+/// ===========================================================================
+:host {
+  @at-root #{&}([on="saturated"][pfe-variant="wind"]) {
+    color: #{pfe-var(link--focus--on-dark)};
+  }
+  @at-root #{&}([on="saturated"][aria-selected="false"][pfe-variant="wind"]:hover) {
+    --pfe-tabs__tab--BorderBottom: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    --pfe-tabs__tab--BorderBottom: 0 !important;
+    --pfe-tabs__tab--BorderBottom: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    --pfe-tabs__tab--BorderBottom: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-var(surface--lightest)} !important;
+  }
+  @at-root #{&}([on="saturated"][aria-selected="true"][pfe-variant="wind"]) {
+    color: var(--pfe-theme--color--surface--lightest, #fff) !important;
+  }
+}
+
+/// ===========================================================================
+/// EARTH TAB VARIANTS
+/// ===========================================================================
+
+:host([pfe-variant="earth"][aria-selected="false"]) {
+  $variantEarth: (
+    BackgroundColor: #f0f0f0,
```

Comment:
same suggestion here, should this override `--pfe-tabs--BackgroundColor` and set the value to `pfe-var(surface--lighter)`
@castastrophe  @alazzara

Reply:
@starryeyez024 That is what's being done here - it's defined in the map and the `pfe-print-local` mixin prints out all the custom properties for you.